README.md

```diff
@@ -148,37 +141,36 @@ kafkaAvro.getConsumer({
 ```
 
 #### Consumer using streams to consume
-
 ```js
-kafkaAvro.getConsumer({
+kafkaAvro.getConsumerStream({
```

Comment:
Are we changing the kafka-avro API?

Reply:
Yes :/

I'm adding a new method because `getReadStream` doesn't exists anymore in the `node-rdkafka`.

So, to keep streamings working, I add this new one that use the new `node-rdkafka` api (check the README.md sample).

Reply:
ok, in that case, we'll have to clearly state the API change on the Changelog marked with **BREAKING CHANGE** so everyone is aware of that when trying to install 1.x

I am assuming that none made the jump to 1.0.0 yet so we won't force a major rev on that